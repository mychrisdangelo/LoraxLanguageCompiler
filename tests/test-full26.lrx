
int main()
{
	tree <int> t(2);
	tree <int> s(2);
	tree <int> m(2);


	t = 1[2[3, 4], 5];
	s = 6[7[8,9], 10[]];
	m = 44[55[],66];

	 t%0%0@ = s@;	// t = 1[2[6, 4], 5];
	 print("t=\n", t , "\ns=\n", s, "\n\n");
	 t = s%0%0; 	//8[null, null]
	 print("t=\n", t , "\ns=\n", s, "\n\n");
	 t = s%0;  		//7[8,9]
	 print("t=\n", t , "\ns=\n", s, "\n\n");
	 t%1%0 = m;
	 print("t=\n", t , "\ns=\n", s, "\n\n");
	 t = s%0%0%0;
	 print("t=\n", t , "\ns=\n", s, "\n\n");
}

