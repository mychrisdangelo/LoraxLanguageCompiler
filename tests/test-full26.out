t=
1[2[6[null,null],4[null,null]],5[null,null]]
s=
6[7[8[null,null],9[null,null]],10[null,null]]

t=
8[null,null]
s=
6[7[8[null,null],9[null,null]],10[null,null]]

t=
7[8[null,null],9[null,null]]
s=
6[7[8[null,null],9[null,null]],10[null,null]]

t=
7[8[null,null],9[44[55[null,null],66[null,null]],null]]
s=
6[7[8[null,null],9[44[55[null,null],66[null,null]],null]],10[null,null]]

t=
null
s=
6[7[8[null,null],9[44[55[null,null],66[null,null]],null]],10[null,null]]

